topic_message_query.go

```diff
@@ -219,8 +222,24 @@ func defaultRetryHandler(err error) bool {
 	code := status.Code(err)
 
 	switch code {
-	case codes.NotFound, codes.ResourceExhausted, codes.Internal, codes.Unavailable:
+	case codes.NotFound, codes.ResourceExhausted, codes.Unavailable:
 		return true
+	case codes.Internal:
+		if RST_STREAM == nil {
+			var err1 error
+			RST_STREAM, err1 = regexp.Compile(".*(rst.stream.*internal.error|internal.error.*rst.stream).*")
```

Comment:
Put this section in an `init` block - https://tutorialedge.net/golang/the-go-init-function/

Reply:
can also use `regexp.MustCompile` which panics so can declare and define `RST_STREAM` in one line.